frontend/public/components/modals/delete-namespace-modal.tsx

```diff
@@ -42,33 +43,39 @@ export const DeleteNamespaceModal: React.FC<DeleteNamespaceModalProps> = ({
   const onKeyUp = (e) => {
     setConfirmed(e.currentTarget.value === resource.metadata.name);
   };
+  const { t } = useTranslation();
+  const { name } = resource.metadata;
 
   return (
     <form onSubmit={onSubmit} name="form" className="modal-content ">
       <ModalTitle className="modal-header">
-        <YellowExclamationTriangleIcon className="co-icon-space-r" /> Delete {kind.label}?
+        <YellowExclamationTriangleIcon className="co-icon-space-r" />{' '}
+        {t('modal~Delete {{label}}?', kind)}
       </ModalTitle>
       <ModalBody>
         <p>
-          This action cannot be undone. It will destroy all pods, services and other objects in the
-          namespace <strong className="co-break-word">{resource.metadata.name}</strong>.
+          <Trans i18nKey="modal~deleteNSWarning">
+            This action cannot be undone. It will destroy all pods, services and other objects in
+            the namespace <strong className="co-break-word">{{ name }}</strong>.
```

Comment:
```suggestion
            This action cannot be undone. It will destroy all Pods, Services and other objects in
            the Namespace <strong className="co-break-word">{{ name }}</strong>.
```